python/smashbox/utilities/__init__.py

```diff
@@ -120,13 +157,12 @@ def run_ocsync(local_folder,remote_folder="",N=None):
 
     for i in range(N):
         t0 = datetime.datetime.now()
-        cmd = config.oc_sync_cmd+' '+local_folder+' '+oc_webdav_url('owncloud',remote_folder)+" >> "+ config.rundir+"/%s-ocsync.step%02d.cnt%03d.log 2>&1"%(reflection.getProcessName(),current_step,ocsync_cnt[current_step])
+        cmd = config.oc_sync_cmd+' '+local_folder+' '+oc_webdav_url(config.oc_root,remote_folder,userNum)+" >> "+ config.rundir+"/%s-ocsync.step%02d.cnt%03d.log 2>&1"%(reflection.getProcessName(),current_step,ocsync_cnt[current_step])
```

Comment:
config.oc_root should not be used here. The first argument of oc_webdav_url is the specification of internal protocol name (http or owncloud) used by owncloudcmd or other client tools